### PR TITLE
Add uv icon

### LINKED ISSUE #11055: Request: uv
### Brand Name

uv

### Website

https://github.com/astral-sh/uv

### Popularity Metric

22.4k stars on GitHub.

### Official Resources for Icon and Color

https://astral.sh/static/SVG/UV.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by Aniganesh**
Preview:
image

**Issue:** closes #11055

**Popularity metric:**
22.4k stars on GitHub.

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Picked hex value is the only one in their logo. It was already "monochrome" (a shade of pink not black).
Did not vectorize logo. It was available as an svg. Had to resize using Inkscape. Ensured all 4 sides are touching the edges.

### CODE REVIEW

**Comment by uncenter** [comment]
>  Ensured all 4 sides are touching the edges.

This is not correct, only two sides should be touching the edges at minimum - lock the height/width ratio and then resize the larger of the two to 24.

**Comment by aniruddha-mithya** [comment]
The problem with that is that the logo is slightly asymetrical. It isn't visible normally but when you open it up in inkscape and blow it up, you can see that none of the edges in that logo are straight. I made the extreme ends of all four sides touch the edge but it still doesn't match the 24x24 requirement. Is it okay to be slightly different from the actual logo? If so I could edit the shape to fit the square. Is there a different way to do it?

**Comment by uncenter** [comment]
> The problem with that is that the logo is slightly asymetrical. It isn't visible normally but when you open it up in inkscape and blow it up, you can see that none of the edges in that logo are straight. I made the extreme ends of all four sides touch the edge but it still doesn't match the 24x24 requirement. Is it okay to be slightly different from the actual logo? If so I could edit the shape to fit the square. Is there a different way to do it?

Like I said, lock the aspect ratio (lock icon button between the height and width fields) and then change the larger of the two dimensions to 24. This makes it so the longer dimension, whichever it is, is at most 24, and the other is that or less. Therefore, it is within 24x24. Then you center and align it, and it should be good to go. You don't need to resize or change or look specifically at what edges are what.

File: icons/uv.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>uTorrent</title><path d="m0 .1058.0504 11.9496.0403 9.5597c.0055 1.3199 1.08 2.3854 2.4 2.3798l9.5596-.0403 5.9749-.0252.6075-.0026c1.316-.0056 2.3799-1.0963 2.3799-2.4123h1.0946v2.3894L24 23.9042 23.8992.005 12.9056.0513l.0463 9.5245v5.9637h-1.9583L11.04 9.584 10.9936.0594Z"/></svg>
```
Comment (uncenter) [inline icons/uv.svg:1]: You have the title here as uTorrent, not uv.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/uv.svg (hunk 1 above)
Comment (Aniganesh) [reply icons/uv.svg]: Fixed.

Comment (Aniganesh) [reply icons/uv.svg]: Would you be open to adding a title-filename check to svglint?

Comment (LitoMore) [reply icons/uv.svg]: We have this. It's just because you're a first-time contributor, some checks will not be run automatically.

Comment (Aniganesh) [reply icons/uv.svg]: I ran the lint command locally before I pushed this commit. Didn't show up. Any idea what I might've done wrong?

Ran `npm run lint` once again after changing the title to uTorrent.
This was the output:
```
> simple-icons@13.14.0 lint /home/aniruddha/Personal Projects/simple-icons
> npm run ourlint && npm run prettierlint && npm run jslint && npm run jsonlint && npm run svglint && npm run wslint


> simple-icons@13.14.0 ourlint
> node scripts/lint/ourlint.js


> simple-icons@13.14.0 prettierlint
> npm run prettier -- --check


> simple-icons@13.14.0 prettier
> prettier --cache --ignore-unknown "**/*.!(js|jsx|mjs|cjs|ts|tsx|mts|cts|svg)" --check

Checking formatting...
All matched files use Prettier code style!

> simple-icons@13.14.0 jslint
> xo

> simple-icons@13.14.0 jsonlint
> node scripts/lint/jsonlint.js


> simple-icons@13.14.0 svglint
> svglint --ci --config svglint.config.mjs icons/*.svg

----------------------------------- Summary -----------------------------------
✓ 3217 valid files.

> simple-icons@13.14.0 wslint
> editorconfig-checker

```
[Screencast from 17-10-24 12:51:00 PM IST.webm](https://github.com/user-attachments/assets/8373580f-7e1c-4c03-b5fe-e7acfdc4cef4)

Comment (LitoMore) [reply icons/uv.svg]: I see. I will file a PR to add this support.

Comment (Aniganesh) [reply icons/uv.svg]: I'm willing to contribute as well. Shall I open a PR?

**Review by LitoMore — APPROVED** [review]

**Pull request merged**